src/elements/name_section.rs

```diff
@@ -97,7 +97,12 @@ impl NameSection {
 					local_names = Some(LocalNameSubsection::deserialize(module, rdr)?);
 				},
 
-				_ => return Err(Error::UnknownNameSubsectionType(subsection_type)),
```

Comment:
Is this error still emitted anywhere? Perhaps would be a cleanup to drop it from the enum.

Reply:
That is a good idea if this is the case.

Reply:
seems like it was only emitted by the code we just removed. 

https://github.com/paritytech/parity-wasm/pull/330